css/components/card.css

```diff
@@ -0,0 +1,145 @@
+/*
+|| Card Css
+*/
+
+.card{
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    min-width: 18rem;
+    max-width: 18rem;
+    background-color: var(--mint-white);   
+    outline: 1px solid var(--light-more-black);
+}
+
+.card-overlay{
+    position: relative;
+}
+
+.card-header{
+    padding: 0.7rem;
+} 
+
+.card-image{
+    max-width: 100%;
+    object-fit: cover;
+    min-height: 10rem;
+}  
+
+.card-title{
+    font-weight: 700;
+    font-size: 1.4rem;
+}
+
+.card-subtitle{
+    font-weight: 300;
+    font-size: 0.9rem;
+    padding: 0.05rem 0rem; 
+    color: var(--light-more-black);
+}
+
+.card-info {    
+    display: flex;
+    justify-content: space-around;
+    font-size: 1rem;
+    font-weight: 400;
+    padding: 0.7rem;
+}
+
+.card-footer {
+    font-size: 1.2rem;
+    padding: 0.3rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.card-item-block { 
+    width: 100%;
+}
+
+.text-btn {
+    color: var(--primary-color);
+    font-weight: 500;
+}
+
+.card-icon {
+    border-radius: 50%;
+    padding: 0.3rem;
+    color: var(--light-blue);
+}
+
+.card-with-badge::before {
+    position: absolute;
+    content: "Off";
```

Comment:
You could place the text part of the badge in the html instead of adding in css.

Reply:
done as you said